### ISSUE TITLE
Rendering Volumes

### ISSUE THREAD

**Issue by LukeLetellier** [ChaosGroup/vray-for-houdini#155]
In the interim help documents, there's a mention of the "V-Ray PhxShaderCache" which apparently is necessary to render volumes with Vray. However, that node doesn't appear to be available anymore in SOPs or in shader Vops. 

If that's the case, what's the current workflow for rendering volumes with Vray?

Thanks,
Luke

**Comment by galagast** [comment]
I just checked, and I also can't seem to find the `PhxShaderCache` node. There used to be one in SOPs.

I also checked my older nightly builds and found that there used to be a `lib` folder the contains an `auraloader.dll` file.

Looking at the latest batch file, this line is supposed to load the library:
```
:: Phoenix Loaders
set VRAY_FOR_HOUDINI_AURA_LOADERS=%LAUNCHER_LOCATION%\vfh_home\vfh_lib
```
So I tried copying over the old `lib` folder to my current build's `vfh_home`, and renamed it to `vfh_lib`.
![image](https://cloud.githubusercontent.com/assets/16534677/22817819/8a5d86e2-efa4-11e6-80cd-ff5de14e69f9.png)
But sadly it didn't work 😞

**Comment by millag** [comment]
This is duplicate of #140. We will update as soon as possible.

**Comment by galagast** [comment]
Thanks! Good to know :)

**Comment by bravehamster** [comment]
https://drive.google.com/open?id=0B1YJxA06WbiZSmFCQmxILUotQ1U

The Feb 9th build works fine, and the vfh_lib folder is present in the launcher location of the plugin.

I had the nodes missing once when I opened one of the old demo scenes from the provided downloads. Try starting with a fresh scene.

**Comment by galagast** [comment]
Hi Gosho, are you on linux? The windows version still seems to be on standby as mentioned in #140.

**Comment by LukeLetellier** [comment]
Thanks for the update, and apologies for rehashing old material. I had searched forum titles for mentions of volumes but hadn't seen anything. I've just now realized that the github filter bar does full searching of threads, not just filtering. 

Perhaps when known issues like this regarding key functionality arise, the wiki could be updated to reflect this? It's been about 11 weeks, so notes like that would be very helpful for new users like myself.

Thanks!
Luke
